generate_questions.py

```diff
@@ -0,0 +1,74 @@
+import os
+from pathlib import Path
+
+# Function to load LOs and K-levels from a text file
+def load_learning_objectives(file_path):
+    data = []
+    with open(file_path, 'r', encoding='utf-8') as file:
```

Comment:
In case of CSV file as an input, this needs to be changed

Reply:
updated